docs/customizations/icon-cache.md

```diff
@@ -0,0 +1,29 @@
+# Icon Cache
+
+Homarr automatically loads icons for many services, though directly links to an external server. To reduce network requests and use homarr offline, a local cache can be used.
```

Comment:
```suggestion
Homarr automatically requests external icons from links. If you do not have access to the internet at all time or want to reduce network traffic, you can use a local cache.
A cache retrieves the pictures from the external source, stores them on your local disk and will tell Homarr to send those pictures instead of the external ones.
```